src/modules/universal-discovery/components/selected-content/selected.content.component.js

```diff
@@ -10,15 +10,14 @@ export default class SelectedContentComponent extends Component {
     constructor(props) {
         super(props);
 
-        this.state = {
-            items: [],
-            isPopupVisible: false,
-        };
-
+        this.hidePopup = this.hidePopup.bind(this);
         this.togglePopup = this.togglePopup.bind(this);
+        this.renderSelectedItem = this.renderSelectedItem.bind(this);
+
+        this.state = { items: [], isPopupVisible: false };
     }
 
-    componentWillReceiveProps(props) {
+    UNSAFE_componentWillReceiveProps(props) {
```

Comment:
I think it would be good idea to start using new lifecycle methods, in this case `getDerivedStateFromProps`.

Reply:
Right, but in this case I just wanted to turn off the linting errors. Feel free to convert this method into a static one.